masonry/src/tree_arena.rs

```diff
@@ -372,6 +372,38 @@ impl<'a, Item> ArenaMutChildren<'a, Item> {
     }
 }
 
+impl<'a, Item> ArenaRef<'a, Item> {
+    pub fn id(&self) -> u64 {
```

Comment:
As this is `pub` I think it could have a `# Panics` doc-comment, that this could panic under certain conditions (same below for `ArenaMut`).

Reply:
The id is never None is that context. Added a comment to document it.